sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/OrderbyDocumentQueryTest.java

```diff
@@ -230,6 +241,49 @@ public void queryOrderByString() throws Exception {
         validateQuerySuccess(queryObservable.byPage(pageSize), validator);
     }
 
+    @Test(groups = { "simple" }, timeOut = TIMEOUT, dataProvider = "sortOrder")
+    public void queryOrderByMixedTypes(String sortOrder) throws Exception {
```

Comment:
>queryOrderByMixedTypes [](start = 16, length = 22)

Does this also work with mulitcolumn order by?

Reply:
Yes, should work with multicolumn orderby. Have a test for it which was checking for unsupported exception. Just fixed it and pushing